### DISCUSSION TITLE
Settings for keeping half the loot

### DISCUSSION THREAD

**Discussion by ByblackSun** [MrHydralisk/Lethal-Company-Save-Our-Loot#2] [Q&A] [answered]
Will this mod make the console lose half the loot if everyone dies? I want to set it up like this, but I don't know if it will work.

**Comment by MrHydralisk** [comment]
Wdym by the "console lose"? Do you mean like after all players died to prevent loosing everything and keep only half of the loot?

**Comment by ByblackSun** [comment]
> Wdym 被「控制台輸了」？你的意思是在所有玩家都死了之後才能防止失去一

Host.  I use google translate.

I mean All players die, keep half of the loot

**Comment by MrHydralisk** [comment, marked as answer]
So then for this you can go one of two ways:
- SaveAllChance = 0, SaveEachChance = 0.5. This will make that each item on your ship have 50% chance of not being lost. Which will result in saving aproximately 50% of your items.
- SaveAllChance = 0, ValueSaveEnabled = true, ValueSavePercent = 0.5. This will make you save loot that in total will cost 50% of scrap value that you had before death. This will try to keep items not by amount, but by it cost.

**Comment by ByblackSun** [comment]
> 那麼為此，您可以採用以下兩種方法之一：
> 
> * 保存所有機會 = 0，保存每個機會 = 0.5。這將使你船上的每件物品都有 50% 的幾率不丟失。這將為您節省約 50% 的物品。
> * SaveAllChance = 0，ValueSaveEnabled = true，ValueSavePercent = 0.5。這將使你節省總共相當於你死前殘值 50% 的戰利品。這將嘗試不按數量保留物品，而是按成本保留物品。

ok 非常好的建議

**Comment by ByblackSun** [comment]
> 因此，您可以採用以下兩種方法之一：
> 
> * 保存所有機會 = 0，保存每個機會 = 0.5。這意味著你其他的每件物品都有 50% 的物品，這樣不會丟失。這將為你節省約 50% 的物品。
> * SaveAllChance = 0，ValueSaveEnabled = true，ValueSavePercent = 0.5。這相當於你節省相當於你死前殘值50%的戰利品。這將嘗試不按數量保留物品，而是按成本保留物品。

SaveAllChance = 0，SaveEachChance = 0.5  ，ValueSaveEnabled = true，ValueSavePercent = 0.5
所以是這樣?

**Comment by MrHydralisk** [comment]
Yes, this should work.
